### PR TITLE
[Trilinos] improving readme

### PR DESCRIPTION
**Pull request by philbucher**

### CODE REVIEW

File: applications/TrilinosApplication/README.md [hunk 1]
```diff
@@ -40,8 +38,14 @@ The Trilinos application also provides __MPI versions of most of the core classe
...
 * [__Epetra__](https://trilinos.org/packages/epetra).
 * [__Aztec__](https://trilinos.org/packages/aztec).
 * [__Amesos__](https://trilinos.org/packages/amesos).
 * [__Teuchos__](https://trilinos.org/packages/teuchos).
+
+### Notes for compilation
+Trilinos is a large project and not all of its packages are being used in Kratos. Check the [configuration file of the CI (Travis)](https://github.com/KratosMultiphysics/Kratos/blob/master/.travis.yml) to see which packages are necessary in order to compile the TrilinosApplication.
+Furthermore it is possible to do a minimal installation of the TrilinosApplication with only using the Epetra package. Using the other packages is enabled by default, but can be disabled with the following flags:
+- *TRILINOS_EXCLUDE_ML_SOLVER*: Setting this flag to `ON` in the configure file will exclude the ML-solver from the compilation
+- *TRILINOS_EXCLUDE_AZTEC_SOLVER*: Setting this flag to `ON` in the configure file will exclude the Aztec-solver from the compilation
```
Comment (jcotela) [inline applications/TrilinosApplication/README.md]: will exclude solvers from the Trilinos AztecOO package.

File: applications/TrilinosApplication/README.md [hunk 2]
```diff
@@ -40,8 +38,14 @@ The Trilinos application also provides __MPI versions of most of the core classe
...
+- *TRILINOS_EXCLUDE_AMESOS_SOLVER*: Setting this flag to `ON` in the configure file will exclude the Amesos-solver from the compilation
```
Comment (jcotela) [inline applications/TrilinosApplication/README.md]: will exclude solvers using features of the Trilinos Amesos package.

File: applications/TrilinosApplication/README.md [hunk 3]
```diff
@@ -40,8 +38,14 @@ The Trilinos application also provides __MPI versions of most of the core classe
...
-* [__Zoltan__](https://trilinos.org/packages/zoltan).
 * [__Epetra__](https://trilinos.org/packages/epetra).
 * [__Aztec__](https://trilinos.org/packages/aztec).
 * [__Amesos__](https://trilinos.org/packages/amesos).
 * [__Teuchos__](https://trilinos.org/packages/teuchos).
+
+### Notes for compilation
+Trilinos is a large project and not all of its packages are being used in Kratos. Check the [configuration file of the CI (Travis)](https://github.com/KratosMultiphysics/Kratos/blob/master/.travis.yml) to see which packages are necessary in order to compile the TrilinosApplication.
+Furthermore it is possible to do a minimal installation of the TrilinosApplication with only using the Epetra package. Using the other packages is enabled by default, but can be disabled with the following flags:
+- *TRILINOS_EXCLUDE_ML_SOLVER*: Setting this flag to `ON` in the configure file will exclude the ML-solver from the compilation
```
Comment (jcotela) [inline applications/TrilinosApplication/README.md]: will exclude the interface to the Trilinos ML solver package.

**Review by jcotela — DISMISSED** [review]
OK overall but I would suggest rewording a bit the description of the compilation flags (for example from AztecOO we do not use a single solver, but a collection of Krylov solvers).

**Comment by philbucher** [comment]
done @jcotela

**Review by jcotela — APPROVED** [review]
